code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -1371,7 +1372,8 @@
 		var/obj/effect/temp_visual/hierophant/blast/B = new(t, user, friendly_fire_check)
 		B.damage = 15 //keeps monster damage boost due to lower damage
 
-
+/obj/item/hierophant_club/admin
```

Comment:
```suggestion
/obj/item/hierophant_club/station
```
Might be nit picky but I think "station" is more appropriate here.
All of our "admin" variants of items have ridiculous stats so naming this one "admin" is almost misleading since it is just the regular item able to function on the station.